op-acceptance-tests/tests/flashblocks/flashblocks_transfer_test.go

```diff
@@ -58,7 +58,7 @@ func TestFlashblocksTransfer(gt *testing.T) {
 	// flashblocks listener
 	flashblocksClient := sys.L2RollupBoost.FlashblocksClient()
 	go func() {
```

Comment:
Spinning up threads in the test feels dangerous - it would be easy for them to be left unresolved (and there's nothing here that guarantees we've actually started listening before the test moves on - if events prior to connecting are dropped that could cause flakiness).